httpd/README.md

```diff
@@ -0,0 +1,17 @@
+[![Docker Repository on Quay](https://quay.io/repository/cybozu/httpd/status "Docker Repository on Quay")](https://quay.io/repository/cybozu/httpd)
+
+httpd container
+===============
+
+This directory provides container image `httpd` and contains its source code.
+
+`httpd` is a simple HTTP server for testing (*not for production*)
+in Kubernetes cluster which is enabled PodSecurityPolicy admission plugin.
```

Comment:
`httpd` is a micro HTTP server that can run in Kubernetes cluster with limited privileges.
Specifically, it runs as a non-root user and does not write to the root filesystem.